tasks/variables-spatialite.yml

```diff
@@ -0,0 +1,26 @@
+- name: Set spatialite_path for Fedora
+  set_fact:
+    openwisp2_spatialite_path: "mod_spatialite"
+  when: ansible_distribution == 'Fedora'
+
+- name: Set spatialite_path (Ubuntu < 18.4 or Debian < 10)
+  set_fact:
+    openwisp2_spatialite_path: "mod_spatialite"
+  when: >
+    (ansible_distribution == 'Ubuntu'
+    and ansible_distribution_version is version_compare('18.04', 'lt'))
+    or (ansible_distribution == 'Debian'
+    and ansible_distribution_version is version_compare('10', 'lt'))
+
+- name: Set spatialite_path (Ubuntu >= 18.4 or Debian >= 10)
+  set_fact:
+    openwisp2_spatialite_path: "mod_spatialite.so"
+  when: >
+    (ansible_distribution == 'Ubuntu' 
+    and ansible_distribution_version is version_compare('18.04', 'ge')) 
+    or (ansible_distribution == 'Debian' and 
+    ansible_distribution_version is version_compare('10', 'ge'))
+
+- name:  Set spatialite_path for Other System (like RedHat & CentOS)
+  set_fact:
+    openwisp2_spatialite_path: false
```

Comment:
this is wrong, it was present in `yum.yml` which was called only when the distribution is redhat based, now this would be always executed, invalidating the previous lines.

Simply remove this because it's not needed anymore.

Reply:
I still wonder why is this here? Am I missing something?